odoo/addons/base/models/assetsbundle.py

```diff
@@ -407,18 +407,28 @@ def get_template(element):
             string = etree.tostring(element, encoding='unicode')
             return string.replace("\\", "\\\\").replace("`", "\\`").replace("${", "\\${")
 
+        names = set()
+        inherit_froms = set()
```

Comment:
Can you use OrderedSet instead ? `set` is not deterministic in Python with string (because of [this](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONHASHSEED)).

Reply:
done